src/assets/images/sistent/copy-button.js

```diff
@@ -0,0 +1,11 @@
+import React from "react"
+
+const CopySvg = ({fillColor}) => {
```

Comment:
there is copy icon might be available from sistent itself ,please use that otherwise pr looks good to me

Reply:
icon added from the protiotype